TodoList/src/app/screens/ToDoList/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import { View, ScrollView, StyleSheet, TextInput, Text } from 'react-native'
+import { connect } from 'react-redux'
+import { Actions } from 'react-native-router-flux'
+
+import { actionCreators } from '../../../redux/actions'
+import Title from '../Title'
+import Footer from '../Footer'
+import List from '../List'
+import Input from '../Input'
+import { styles } from './styles.js'
+
+class ToDoApp extends Component {
+    addItem = (item) => {
+    const {dispatch} = this.props
+    dispatch(actionCreators.addItem(item))
+  }
+  removeItem = (index) => {
+    const {dispatch} = this.props
+    dispatch(actionCreators.removeItem(index))
+  }
+  toggleItem = (index) => {
+    const {dispatch} = this.props
+    dispatch(actionCreators.toggleItem(index))
+  }
+  removeCompleted = () => {
+    const {dispatch} = this.props
+    dispatch(actionCreators.removeCompleted())
+  }
+  
+render() {
+  const {items} = this.props
+  return (
+    <View style={styles.container}>
+      <Title> Todo List </Title>
+        <Input
+          placeholder={'Enter an item!'}
+          onSubmit={this.addItem}
+        />
+        <View style={styles.divider}/>
```

Comment:
You don't need these Views. Just add style to the other components to separate them